models/comments.models.js

```diff
@@ -0,0 +1,21 @@
+const db = require('../db/connection')
+
+function fetchComments(article_id){
+    return db.query(
+        `
+        SELECT
+            comment_id,
+            votes,
+            created_at,
+            author,
+            body,
+            article_id
+        FROM comments
+        WHERE article_id = $1
+        ORDER BY created_at DESC;
+        `,
+        [article_id]
+    )
+}
```

Comment:
You don't seem to be returning rows here either? We also need to handle two possibilities in here: a 404 as article not found and a 200 when comments are not there yet.